File: Kata 8/EvenOrOdd/EvenOrOdd.py
"""
Kata Level: 8.

Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
"""

def even_or_odd(number):
    return "Even" if number % 2 == 0 else "Odd"

print(even_or_odd(0))
print(even_or_odd(10))
print(even_or_odd(1))
print(even_or_odd(13))

File: Kata 8/EvenOrOdd/EvenOrOdd.cs
/* Kata Level: 8.
 * 
 * Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
 */

using System;

namespace Solution
{
  public class SolutionClass
  {
    public static string EvenOrOdd(int number)
    {
      return number % 2 == 0 ? "Even" : "Odd";
    }

    public static void Main(string[] args)
    {
      Console.WriteLine(EvenOrOdd(0));
      Console.WriteLine(EvenOrOdd(10));
      Console.WriteLine(EvenOrOdd(3));
      Console.WriteLine(EvenOrOdd(55)); 
    }
  }
}